src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -102,11 +102,6 @@
 
         gb.withRateLimitHandler(CUSTOMIZED);
         OkHttpClient client = new OkHttpClient().setProxy(getProxy(defaultIfBlank(apiUrl, GITHUB_URL)));
-        client.setCache(GitHubClientCacheOps.toCacheDir().apply(config));
-        if (config.getClientCacheSize() > 0) {
-            Cache cache = toCacheDir().apply(config);
-            client.setCache(cache);
-        }
```

Comment:
> That's why github plugin configuration allows disabling it. But you are using your custom connector.

The configuration is shared, so I guess disabling cache at configuration would disable it here too (before this PR, now there is no cache).